### ISSUE TITLE
Pepys database connection issue

### ISSUE THREAD

**Issue by IanMayo** [debrief/debrief#5122]
Analysts are pulling data into Debrief from Pepys, using the Pepys Import tool.  They're in version 3.1.30, dated `2021/05/30`

The `Test connection` function works, but an error is thrown when the `Apply` button is clicked for filtering.

[exact message to follow]

I've looked at the error log.  A "normal" series of database import tracking logging message are shown, including a reference to using the `Fast Mode` algorithm.

At the top of the error log is a NPE in `OSUtils.getInputStreamResource (OSUtils.java#56)`

Note: we know that the Pepys database is up & running - since it could be verified in PGAdmin.  
Note: there is a current issue with `Pepys-Admin`, which is showing an Alembic migration error.  But, Debrief doesn't use Alembic.

**Comment by IanMayo** [comment]
Robin has usefully pointed out that one of the missing migrations is the creation of the Stored Procedures that offer simpler connections and faster data transfer.  
_If_ Debrief is trying to use the new stored procedures then that would cause an issue.  We may be able to reproduce the issue by (temporarily) changing the name of the Stored procedure in the AcuGis instance.

**Comment by IanMayo** [comment]
Ok. There's a strong chance the issue here is down to the stored procedures being missing from Pepys.

Debrief and pepys have got out of sync - so the `Stored procedures` used by Debrief aren't present.

But, I was able to do some further diagnostics today.  I switched off `Use stored procedures` in `Debrief-specific settings`.  I was able to filter the data, but when I tried to load tracks there was a long delay, then:
```
The column name reference was not found in the ResultSet
```

The lowest level Debrief code to this was at `StateFastMode.retrieveObject` in `StateFastMode.java#110`

This was where the `Speed` value was retrieved.
